demisto_sdk/commands/format/update_generic_json.py

```diff
@@ -0,0 +1,152 @@
+import os
+import json
+import yaml
+from demisto_sdk.commands.common.tools import print_color, LOG_COLORS
+from demisto_sdk.commands.common.hook_validations.structure import StructureValidator
+
+
+class BaseUpdateJSON:
+    """BaseUpdateJSON is the base class for all json updaters.
+
+        Attributes:
+            input (str): the path to the file we are updating at the moment.
+            output (str): the desired file name to save the updated version of the YML to.
+            json_data (Dict): YML file data arranged in a Dict.
+    """
+
+    DEFAULT_JSON_VERSION = -1
+    NEW_FILE_DEFAULT_FROMVERSION = '5.0.0'
+    OLD_FILE_DEFAULT_FROMVERSION = '1.0.0'
```

Comment:
you have defined it in a different file.
consider putting all constants in the same file, e.g in `constants.py`